src/evaluation_instruments/_evaluation.py

```diff
@@ -214,16 +215,25 @@ def post_process_default(self, sample_ix, openai_json: dict) -> tuple[dict, Toke
             as well as the usage information from response['usage'].
         """
         ix = 0  # assume N=1
+        try: #  Many providers have their own response objects, try to convert
+            openai_json = openai_json.json()
+        except AttributeError:
+            pass
+
+        if isinstance(openai_json, str):
+            openai_json = json.loads(openai_json)
+
         try:
             raw_content = openai_json["choices"][ix]["message"]["content"]
-
             # Assume no nesting {}
             response = json.loads(raw_content[raw_content.find("{") : raw_content.find("}") + 1])  # noqa: E203
         except Exception:
             logger.info(f"Failed to parse {sample_ix} response content as JSON.")
             response = {}
 
         usage = openai_json.get("usage", {"completion_tokens": 0, "prompt_tokens": 0, "total_tokens": 0})
```

Comment:
Here is what I needed to do to make this work for litellm - 

```suggestion
        if type(openai_json) is not dict:  #litellm response object is not a dict
            usage = openai_json["usage"].dict
        else:
            usage = openai_json.get("usage", {"completion_tokens": 0, "prompt_tokens": 0, "total_tokens": 0})
```